### PR TITLE
Release notes for Vsts 2.0.10

### PR DESCRIPTION
**Pull request by Yiming-Jia**

### CODE REVIEW

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.10 - March, 21 2020
+* Supported to trim key prefixes before set key-values into environment variables.
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: We could rephrase this as `Added support to trim one or more prefixes from the keys of all key-values before these are set as environment variables`

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+* Supported to use Azure Key Vault reference.
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Suggested wording for this one : `Added support to resolve key vault references`

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
 
+### v2.0.10 - March, 21 2020
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: What is the versioning scheme we use for the VSTS Task? I would generally expect addition of new features to bump up the `Minor` version instead of the `Patch` version.

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
```
Comment (abhilasharora) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: Could we add one-line space between the text and the image?

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 1 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: `Added support to trim key prefixes before setting as pipeline variables.`

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 2 above)
Comment (abhilasharora) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Added support **to resolve** instead of **of resolving**

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: `Added support for doing`

File: releaseNotes/AzureDevOpsPipelineExtension.md (hunk 4 above)
Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Sure

File: releaseNotes/AzureDevOpsPipelineExtension.md [hunk 5]
```diff
@@ -1,7 +1,12 @@
 ## Azure App Configuration Extension release notes
 
-The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: ![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+The Azure App Configuration extension in Azure DevOps pipeline has been published to [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task&ssr=false#overview). The version of the Azure App Configuration task being used will print to the console when the task is executed: 
 
+![sample](pictures/AzureDevOpsExtensionVersionSample.PNG)
+
+### v2.0.10 - March, 21 2020
+* Added support for trimming key prefixes before setting as pipeline variables.
+* Added support for resolving Key Vault references and set as pipeline secrets.
```
Comment (zhenlan) [inline releaseNotes/AzureDevOpsPipelineExtension.md]: ...and *setting* as...

Comment (Yiming-Jia) [reply releaseNotes/AzureDevOpsPipelineExtension.md]: Setting as
